proto/viam/service/worldstatestore/v1/world_state_store.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+package viam.service.worldstatestore.v1;
+
+import "common/v1/common.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+
+option go_package = "go.viam.com/api/service/worldstatestore/v1";
+option java_package = "com.viam.service.worldstatestore.v1";
+
+
+service WorldStateStoreService {
+  // ListWorldStates returns all world state object uuids
+  rpc ListWorldStateUUIDs(ListWorldStateUUIDsRequest) returns (ListWorldStateUUIDsResponse) {}
+
+  // GetWorldState returns a world state object by uuid
+  rpc GetWorldStateObject(GetWorldStateObjectRequest) returns (GetWorldStateObjectResponse) {}
+
+  // DoCommand sends/receives arbitrary commands
+  rpc DoCommand(common.v1.DoCommandRequest) returns (common.v1.DoCommandResponse) {
```

Comment:
How are we ever supposed to be able to add things to this?

Reply:
I am not sure what you are asking, but this how all `DoCommand`s are defined for components and services. Did I miss something? Or Is there something specific you think we should be adding for this service?

Examples:

```
  // DoCommand sends/receives arbitrary commands
  rpc DoCommand(common.v1.DoCommandRequest) returns (common.v1.DoCommandResponse) {
    option (google.api.http) = {post: "/viam/api/v1/component/button/{name}/do_command"};
  }
 
 ...
 
   // DoCommand sends/receives arbitrary commands
  rpc DoCommand(common.v1.DoCommandRequest) returns (common.v1.DoCommandResponse) {
    option (google.api.http) = {post: "/viam/api/v1/service/navigation/{name}/do_command"};
  }
  ```

Reply:
That was in the scope:

>  // Add / remove objects de-scoped due to not being needed yet.
  // Existing examples above involve a module implementation of this service 
  // querying other resources and internally adding / removing objects.